app/frontend/src/features/auth/login/Login.tsx

```diff
@@ -42,32 +51,75 @@ export default function Login() {
   return (
     <>
       {authenticated && <Redirect to={redirectTo} />}
-      <Box className={authClasses.backgroundBlurImage}></Box>
-      <Box className={authClasses.page}>
-        <AuthHeader>Welcome back!</AuthHeader>
-        {error && (
-          <Alert className={authClasses.errorMessage} severity="error">
-            {error}
-          </Alert>
-        )}
-        <LoginForm />
-        {/* <Divider>Or</Divider>  not yet available: https://next.material-ui.com/components/dividers/ */}
-        {/* Disabled for beta:
-        <Divider classes={{ root: authClasses.divider }} flexItem />
-        <MuiButton className={classes.facebookButton}>
-          Login with Facebook
-        </MuiButton>
-        <MuiButton className={classes.googleButton}>
-          Login with Google
-        </MuiButton>
-        */}
-        <Typography className={classes.signUp}>
-          No account yet?{" "}
-          <Link className={classes.signUpLink} to={signupRoute}>
-            Sign up
-          </Link>
-        </Typography>
-      </Box>
+      <Hidden mdUp>
+        <div className={authClasses.backgroundBlurImage}></div>
+        <div className={authClasses.page}>
+          <AuthHeader>Welcome back!</AuthHeader>
+          {error && (
+            <Alert className={authClasses.errorMessage} severity="error">
+              {error}
+            </Alert>
+          )}
+          <LoginForm />
+          {/* <Divider>Or</Divider>  not yet available: https://next.material-ui.com/components/dividers/ */}
+          {/* Disabled for beta:
+          <Divider classes={{ root: authClasses.divider }} flexItem />
+          <MuiButton className={classes.facebookButton}>
+            Login with Facebook
+          </MuiButton>
+          <MuiButton className={classes.googleButton}>
+            Login with Google
+          </MuiButton> */}
+          <Typography className={classes.signUp}>
+            No account yet?{" "}
+            <Link className={classes.signUpLink} to={signupRoute}>
+              Sign up
+            </Link>
+          </Typography>
+        </div>
+      </Hidden>
+      <Hidden smDown>
+        <div className={authClasses.page}>
+          <header className={authClasses.header}>
+            <div className={authClasses.logo}>Couchers.org</div>
+            <Typography className={classes.signUp}>
+              No account yet?{" "}
+              <Link className={classes.signUpLink} to={signupRoute}>
+                Sign up
+              </Link>
+            </Typography>
+          </header>
+          <div className={authClasses.content}>
+            <div className={authClasses.introduction}>
+              <Typography classes={{ root: authClasses.title }} variant="h1">
+                {INTRODUCTION}
+              </Typography>
+              <Typography classes={{ root: authClasses.subtitle }} variant="h2">
+                {COLLABORATIVE}
```

Comment:
Avoid this kind of constant name for longer text, use LOGIN_TAGLINE_1 and 2, or something more semantic

Reply:
Not mine, but changed to `INTRODUCTION_TITLE` and `INTRODUCTION_SUBTITLE`